### PR TITLE
[4.0] Rename method arguments to use camelCase casing

### PR DESCRIPTION
**Pull request by SharkyKZ**
Partial Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30988.

### Summary of Changes

Renames some method arguments to use camelCase casing.

### Testing Instructions

Thorough code review / Joomla works like before.

### Documentation Changes Required

No.

### CODE REVIEW

File: administrator/components/com_finder/src/Indexer/Language/El.php [hunk 1]
```diff
@@ -832,23 +832,23 @@ public function stem($token)
...
-		return $this->toLowerCase($token, $w_CASE);
+		return $this->toLowerCase($token, $wCase);
 	}
 
 	/**
-	 * Converts the token to uppercase, suppressing accents and diaeresis. The array $w_CASE contains a special map of
+	 * Converts the token to uppercase, suppressing accents and diaeresis. The array $wCase contains a special map of
 	 * the uppercase rule used to convert each character at each position.
 	 *
 	 * @param   string  $token    Token to process
```
Comment (Quy) [inline administrator/components/com_finder/src/Indexer/Language/El.php]:
```suggestion
	 * @param   string  $token   Token to process
```

File: administrator/components/com_finder/src/Indexer/Language/El.php [hunk 2]
```diff
@@ -904,73 +904,73 @@ protected function toUpperCase($token, &$w_CASE)
...
 		return $newToken;
 	}
 
 	/**
-	 * Converts the suppressed uppercase token back to lowercase, using the $w_CASE map to add back the accents,
+	 * Converts the suppressed uppercase token back to lowercase, using the $wCase map to add back the accents,
 	 * diaeresis and handle the special case of final sigma (different lowercase glyph than the regular sigma, only
 	 * used at the end of words).
 	 *
 	 * @param   string  $token   Token to process
```
Comment (Quy) [inline administrator/components/com_finder/src/Indexer/Language/El.php]:
```suggestion
	 * @param   string  $token  Token to process
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 90e03d13d430773fa2291d2fad458b488e2b6e10<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31181">issues.joomla.org/tracker/joomla-cms/31181</a>.</sub>

**Review by bembelimen — APPROVED** [review]

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 90e03d13d430773fa2291d2fad458b488e2b6e10<br /><br />Tested on clean install. All functionality works as before PR.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31181">issues.joomla.org/tracker/joomla-cms/31181</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31181">issues.joomla.org/tracker/joomla-cms/31181</a>.</sub>

**Comment by chmst** [comment]
Thanks!

**Pull request merged**